src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -109,6 +111,15 @@ public OpenLoadFlowParameters setDcUseTransformerRatio(boolean dcUseTransformerR
         return this;
     }
 
+    public boolean isUpdateFlows() {
```

Comment:
I am not sure to understand if we really need this kind of parameters. In which use case are we doing not to update flows ?

Reply:
At this level (external API), we always wang to update flows so we should remove it here.
Internally (in AC/DC engines) we sometimes don't want to update flows.